css/slyles.css

```
@@ -0,0 +1,295 @@
+body {
+  background-color: #F2F2F2;
+  color: #333333;
+  font-family: "PT Sans", sans-serif;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.page-header {
+  width: 1200px;
+  background-color: #FFFFFF;
+}
+
+.main-navigation {
+  list-style: none;
+}
+
+.main-navigation-link {
+  color: #000000;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  text-decoration: none;
+}
+
+.navigation-user {
+  list-style: none;
+}
+
+
+.favourites-number {
+  color: #FFFFFF;
+  font-size: 10px;
+  line-height: 10px;
+  background: #7DB54F;
+}
+
+.navigation-user-want {
+  background-color: #756157;
+  color: #FFFFFF;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.page-main {
+  width: 1200px;
+  background-color: #FFFFFF;
+  text-align: center;
+}
+
+.hero {
+  height: 485px;
+  background: url(../images/hero-background-bottom.svg) bottom left no-repeat,
+              url(../images/hero-background.jpg) top left no-repeat;
+}
+
+.bold {
+  color: #000000;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.advantages-list {
+  list-style: none;
+}
+
+.title {
+  color: #000000;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-transform: uppercase;
+}
+
+.advantages-list-item {
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.advantages-list li:first-child, .advantages-list li:last-child {
+  color: #FFFFFF;
+  background-color: #82B3D3;
+}
+
+.advantages-list li:first-child .title, .advantages-list li:last-child .title {
+  color: #FFFFFF;
+}
+
+.advantages-list li:nth-child(3), .advantages-list li:nth-child(5) {
```

здесь можно заменить на вот такую конструкцию:

```suggestion
.advantages-list li:nth-child(2n+1) {
```

Это будет означать, что, мы берем все нечетные li.
Чтобы перебить специфичность и стилизовать first-child, нужно опустить объявление li:first-child под этот блок